applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -1,4 +1,19 @@
 import KratosMultiphysics
+import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
+import math
+
+def DotProduct(A,B):
+    result = 0
+    for i, j in zip(A,B):
+        result += i*j
+    return result
+
+def CrossProduct(A, B):
```

Comment:
same

Reply:
I could also not find the cross product implemented in c++

Reply:
By the way, this is just done once so I do not think it is so important. Specially when we are anyway going to migrate to c++.

Reply:
ok

BTW it is implemented in the MathUtils